sdk-python/livescoreapi.py

```diff
@@ -10,11 +10,50 @@
 
 
 class livescores:
+    """
+    A class used to get livescores. 
+    ...
+
+    Attributes
+    ----------
+    key : str
+        an alphanumeric string used to access data from the LiveScore API
+    secret : str
+        an alphanumeric string used to access data from the LiveScore API
+    
+    Methods
+    -------
+    _get_html_file(link)
+    
+    get_all()
+    
+    """
+    def __init__(self,key,secret):
+        self.key = key 
```

Comment:
What happens if the key that is provided is actually empty? We also have these rules that we can add as a check: the api key is always 16 characters long and api secret is always 32 characters long. And we need to clean the trailing spaces in front and in the end.